test/liquid_voting_web/absinthe/mutations/create_delegation_test.exs

```diff
@@ -191,4 +193,129 @@ defmodule LiquidVotingWeb.Absinthe.Mutations.CreateDelegationTest do
       assert details == %{delegate_id: ["can't be blank"]}
     end
   end
+
+  describe "create new global delegation for delegator with existing global delegation" do
+    test "overwrites existing global delegation" do
+      query = """
+      mutation {
+        createDelegation(delegatorEmail: "#{@new_delegator_email}", delegateEmail: "#{
+        @new_delegate_email
+      }") {
+          delegator {
+            email
+            name
+          }
+          delegate {
+            email
+            name
+          }
+        }
+      }
+      """
+
+      {:ok, %{data: %{"createDelegation" => original_delegation}}} =
+        Absinthe.run(query, Schema, context: %{organization_id: @organization_id})
+
+      assert original_delegation["delegator"]["email"] == @new_delegator_email
+      assert original_delegation["delegate"]["email"] == @new_delegate_email
+
+      query = """
+      mutation {
+        createDelegation(delegatorEmail: "#{@new_delegator_email}", delegateEmail: "#{
+        @new_other_delegate_email
+      }") {
+          delegator {
+            email
+            name
+          }
+          delegate {
+            email
+            name
+          }
+        }
+      }
+      """
+
+      {:ok, %{data: %{"createDelegation" => modified_delegation}}} =
+        Absinthe.run(query, Schema, context: %{organization_id: @organization_id})
+
+      assert modified_delegation["delegator"]["email"] == @new_delegator_email
+      assert modified_delegation["delegate"]["email"] == @new_other_delegate_email
+
+      query = """
+      query {
+        delegations {
+          id 
+        }
+      }
+      """
+
+      {:ok, %{data: %{"delegations" => delegations}}} =
+        Absinthe.run(query, Schema, context: %{organization_id: @organization_id})
+
+      assert Enum.count(delegations) == 1
```

Comment:
I don't think we need to test the delegation count here, since we're already testing it on the LiquidVoting.Delegations layer.

Graphql tests (and most API layer tests) are really just to meant to assert what a client consuming the API should expect. Database counts aren't their concern, unless they're part of the payload being sent.

Reply:
One thing that could be tested as an API, but I'm not too sure at this point, is sending out a query for the previous delegation, to assert it doesn't exist anymore and the client doesn't get anything back. What do you think?

Reply:
The previous delegation exists in modified form. Thus, the client would get the modified delegation back (with a query using `delegationId`).

Reply:
I'm thinking of each version of the delegation as separate delegations, as it would be from a user's point of view, even if behind the scenes we're modifying the same record on the db. 

With a query using an old delegate as param, we'd be asserting that a query for the old delegation doesn't return anything, and it'd be more explicit than counting the delegations for an organization.

Reply:
I kind of understand what you mean, but I'm confused by the fact that the only way to query a specific delegation (old or new) is by id (at absinthe level, for user). So, the only query possible to check if an 'old' delegation exists (for same delegate or not) is using the id, which will be used by the 'new' updated delegation (and thus it exists, even if it has a different delegate).

Reply:
Maybe we could do it by querying delegations for an org, and returning delegate/delegator ids, etc. in the result, and asserting that a specific combo of delegation specific values does not exist after an update to a delegation?

Reply:
I see what you mean - the resolver for querying a single delegation only accepts an id right now! I had Delegations.get_delegation!/3 in mind, not the resolver. In this case I think the way you're doing it now is the best solution, I don't think asserting the missing values would add much. I'm going to mark this as resolved and open an issue for updating the resolver, I think we need querying by delegator and delegate at that level as well.